src/server/game/Server/Packets/QuestPackets.h

```diff
@@ -22,6 +22,7 @@
 #include "ItemPacketsCommon.h"
 #include "QuestDef.h"
 #include "ObjectGuid.h"
+#include "ObjectMgr.h"
```

Comment:
Massive objection.
Forward declare `PlayerChoice` and use a pointer to it in packet class.